src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { PreparedPost } from '../../Types/Types';
+import './PostInfo.scss';
+
+type Props = {
+  onePost: PreparedPost;
+};
+
+export const PostInfo:React.FC<Props> = (props) => {
```

Comment:
make destructuring
and just `post` better name than `onePost`

```suggestion
export const PostInfo:React.FC<Props> = ({ post }) => {
```